dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java

```diff
@@ -724,7 +724,7 @@ private int createCommand(CommandType commandType, long processDefineCode, TaskD
                 return createComplementCommandList(schedule, runMode, command, expectedParallelismNumber,
                     complementDependentMode);
             } catch (CronParseException cronParseException) {
-                // this just make compile happy, since we already validate the cron before
+                // this just make compile happen, since we already validate the cron before
```

Comment:
Not sure what this comment means here. It makes developers confused. If we do not need this, maybe we could just delete it.

Reply:
I just saw a spelling error here, and I was confused at the time, but I don't know if it should be deleted, and I want to see other reviewers' opinions

Reply:
I think we can delete it

Reply:
Oh, I think it add this comment for specific case, The init idea is when even `CronParseException ` throw from function `createComplementCommandList ` or not, we should think it pass.

So maybe this sentence mean, `we add this catch to pass compiler pass, because we throw exception in function CronParseException`.

In this case I think we should remove the error log in L728 but keep this comment, WDYT @EricGao888

Reply:
I got it. Thx @zhongjiajie  for the explanation. We could remove the error logging and update the comment as `We catch the exception here just to make compiler happy, since we have already validated the schedule cron expression before` for better comprehension. @fuchanghai

Reply:
Are you sure you don't replace happy with happen

> I got it. Thx @zhongjiajie for the explanation. We could remove the error logging and update the comment as `We catch the exception here just to make compiler happy, since we have already validated the schedule cron expression before` for better comprehension. @fuchanghai